tests/functional/spec/features/repeating_sections/repeating_sections_with_hub_and_spoke.spec.js

```diff
@@ -145,7 +145,7 @@ describe("Feature: Repeating Sections with Hub and Spoke", () => {
 
     it("When the user views the summary for a repeating section, Then the page title is shown", () => {
       $(HubPage.summaryRowLink("personal-details-section-2")).click();
-      expect(browser.getTitle()).to.equal("… - Hub & Spoke");
+      expect(browser.getTitle()).to.equal("… - Hub & Spoke - Hub & Spoke");
```

Comment:
I think the `_get_safe_page_title` method in the SectionSummaryContext handler needs to be updated so that it doesn't append the schema title twice

Reply:
Thanks, I just removed it and fixed some tests👍